src/backend/api/Fusion.Resources.Domain/Models/QueryDepartment.cs

```diff
@@ -25,4 +25,5 @@ public QueryDepartment(string departmentId, string? sectorId)
         public List<FusionPersonProfile>? DelegatedResourceOwners { get; set; }
         public bool IsTracked { get; set; } = false;
```

Comment:
Is IsTracked in use?

Reply:
Dunno, not relevant for this pr.. could add new work item to refactor :)